llvm/tools/llvm-cas-test/Config.def

```diff
@@ -0,0 +1,18 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Define CONFIG before including.
+//
+// #define CONFIG(NAME, TYPE, DEFAULT_VAL, MIN_VAL, MAX_VAL, OPT_NAME, OPT_DESC)
```

Comment:
The includes are using the order [default, max, min], but in this comment it is [default, min, max]. Most of the values below appear to be [default, min, max]